content/en/universal_service_monitoring/setup.md

```diff
@@ -906,6 +906,21 @@ agents:
         - name: DD_SERVICE_MONITORING_CONFIG_TLS_ISTIO_ENABLED
           value: "true"
 ```
+
+If your cluster is using the Bottlerocket Linux distribution for its nodes, add the following to your values file:
+
+```
+agents:
+  containers:
+    systemProbe:
+      securityContext:
+        seLinuxOptions:
+          user: "system_u"
+          role: "system_r"
+          type: "spc_t"
+          level: "s0"
+```
+
```

Comment:
why is it under istio section?

Reply:
let's move it to a different section, as it can affect other components of system-probe and there is nothing unique for istio

Reply:
Moved it below the instructions for GCE